.github/workflows/build.yml

```diff
@@ -34,16 +34,14 @@ jobs:
 
       - uses: cloudflare/wrangler-action@v3
         with:
-          wranglerVersion: "3.57.0"
+          wranglerVersion: "3.63.1"
           apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
           accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
           secrets: |
-            WEBHOOK_PROXY_URL
-            WEBHOOK_SECRET
             APP_ID
-            PRIVATE_KEY
+            APP_PRIVATE_KEY
+            WEBHOOK_SECRET
```

Comment:
Debating on if it makes sense to prefix this with APP since it is defined in the GitHub App settings.

Reply:
I think in this case it makes it a bit clearer that it is related to the App, because `PRIVATE_KEY` could be anything (we added a `UBIQUIBOT_PRIVATE_KEY` in other repos for example). Can still revert of course.